src/REveMainWindow.cc

```diff
@@ -111,6 +115,10 @@ void REveMainWindow::showNodesByName(TGeoNode* n, const std::string& str, bool o
                   FrontTracker_gdmltag = j;
                   
                  }
+                std::string CRVstr("CRS"); // ("CRSScintillatorBar_1_0");
+                if (name.find(CRVstr)!= std::string::npos){
```

Comment:
I added something similar, can you reuse the bool I made?

Reply:
Hi Sophie, yes I think I can.

Reply:
Hi Sophie, it's fixed.